src/test/java/StringManipulationTest.java

```diff
@@ -19,65 +18,109 @@ public void tearDown() {
         manipulatedstring = null;
     }
 
+    /**
+     * Check whether a simple string of space-separated words is counted correctly.
+     */
     @Test
     public void testCount1() {
         manipulatedstring.setString("This is my string");
         int length = manipulatedstring.count();
         assertEquals(4, length);
     }
 
+    /**
+     * Check whether an empty string is correctly counted as zero.
+     */
     @Test
     public void testCount2() {
-        fail("Not yet implemented");
+        manipulatedstring.setString("");
+        assertEquals(0, manipulatedstring.count());
     }
 
+    /**
+     * Check whether a single letter string is correctly counted as one.
+     */
     @Test
     public void testCount3() {
-        fail("Not yet implemented");
+        manipulatedstring.setString("A");
+        assertEquals(1, manipulatedstring.count());
     }
```

Comment:
You are testing a single character string. This is another good edge case. Have you considered testing a string with only one space.